sync_radar_data_globus.py

```diff
@@ -171,8 +172,37 @@ def synchronize(self):
                 pass
             else:
                 pass
-            listing = self.transfer_client.operation_ls(self.mirror_uuid, path=listing_path,
-                                                        filter=listing_pattern)
+            print("Listing path: {path} on endpoint: {endpoint} with pattern: {pattern}".format(
+                path=listing_path, endpoint=self.mirror_uuid, pattern=listing_pattern))
+            listing_succeeded = False
+            operation_ls_retries = 10 
```

Comment:
Any reason for 10?

Reply:
it was the number suggested by Sergiy, but I've found that 15 is more reasonable, so I've changed it to 15. I'll add comment